docs/.vitepress/theme/components/DbToolsBundleFlavorSwitcher.vue

```diff
@@ -0,0 +1,88 @@
+<script setup lang="ts">
+import { useFlavour } from './flavour';
+const { flavour } = useFlavour()
+const id = Math.random().toString(36).substring(2, 6);
```

Comment:
Why a random id here, and not a fixed one?

Reply:
I had issue when this component is present twice in the page

Reply:
Does this happen often? It seems weird to have it more than once.